packages/server/package.json

```diff
@@ -38,7 +38,7 @@
     "chai": "1.10.0",
     "chalk": "2.4.2",
     "check-more-types": "2.24.0",
-    "chokidar": "3.2.2",
+    "chokidar": "3.5.1",
```

Comment:
Do we need to bump chokidar? Last time we did, it introduced some unexpected flake to CI. Unless we *need* too, I think deps should be bumped in their own PR. If we need 3.5.1, though, that's different.

Reply:
I found that cypress was only watching the first set of changes.
Closing the project and re-opening it was not restarting the watchers properly.
There was a fix done specifically fo that in v3.3 but it broke a lot more than it fixed.
Updating chokidar to 3.5.1 seemed to fix all the problems I had.

https://github.com/paulmillr/chokidar#changelog